docs/set.md

```diff
@@ -0,0 +1,33 @@
+# Set
+
+The set command supports the following sub-commands:
+
+* [Recovery-profile](#recovery-profile)
+
+## Recovery-profile
+
+This command is used choose/modify the [mclock](https://docs.ceph.com/en/quincy/rados/configuration/mclock-config-ref/#mclock-profiles-definition-and-purpose) scheduler profile that is being used during recovery.
+
+We can use the following built-in profile types:
+
+- [balanced](https://docs.ceph.com/en/quincy/rados/configuration/mclock-config-ref/#balanced-default) (default)
+- [high_client_ops](https://docs.ceph.com/en/quincy/rados/configuration/mclock-config-ref/#high-client-ops)
+- [high_recovery_ops](https://docs.ceph.com/en/quincy/rados/configuration/mclock-config-ref/#high-recovery-ops)
```

Comment:
Should these link to RHCS docs?

Reply:
I was not able to point url to each individual topics